config/jobs/kubernetes/sig-node/dra-canary.yaml

```diff
@@ -254,3 +254,97 @@ presubmits:
           requests:
             cpu: 2
             memory: 9Gi
+
+  - name: pull-kubernetes-node-e2e-crio-cgrpv1-dra-kubetest2-canary
```

Comment:
You shouldn't have to create "kubetest2-canary". Just modify the existing "canary" job.

That conflicts with other, concurrent changes to canary, like my timeout changes. In practice, most of the time we are only going to try one experimental change at a time, so that should be okay.

Let me wrap up my experimental work by merging it into the main jobs...

Reply:
Once https://github.com/kubernetes/test-infra/pull/34166 is merged, you can make whatever change you want for the canary jobs inside if sections, similar to how I did it in https://github.com/kubernetes/test-infra/pull/34150.

Reply:
See https://github.com/kubernetes/test-infra/pull/34254